multigeiger/userdefines-example.h

```diff
@@ -63,3 +63,20 @@
 // 0x2A38: Heart Rate Sensor Position --> sends TUBE_TYPE
 // 0x2A39: Heart Rate Control Point --> allows to reset "energy expenditure", as required by service definition
 #define SEND2BLE false
+
+// Play an alarm sound when radiation level is too high?
+// Activates when either accumulated dose rate reaches the set threshold (s. below)
+// or when the current dose rate is higher than the accumulated dose rate by the set factor (s. below)
```

Comment:
is "s." really an abbreviation of "see"?

guess i would rather keep "see", saving 1 char is a bit pointless. :-)

Reply:
please add a dot at the end of the sentences.

Reply:
"s." is often used as abbreviation, but indeed doesn't matter here. Added dots.